src/include/stat.h

```diff
@@ -100,7 +102,7 @@ __wt_stats_aggregate(void *stats_arg, int slot)
     int i;
 
     stats = (int64_t **)stats_arg;
-    for (aggr_v = 0, i = 0; i < WT_COUNTER_SLOTS; i++)
+    for (aggr_v = 0, i = 0; i < WT_STAT_CONN_COUNTER_SLOTS; i++)
```

Comment:
Is it the case that this can only get called for conn stats? If so, I think this should be renamed to make that clear.